### PR TITLE
fix(service-portal): Layout fixes after testing

### PR DESCRIPTION
**Pull request by Tryggvig**
# Fixes after layout design review 

*https://www.notion.so/751852d7f3c74a82b72c388acd341a1d?v=e7cc864fa18f4fab83cf26b3b6009d18&pvs=4

## Checklist:

- [x] I have performed a self-review of my own code
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] Formatting passes locally with my changes
- [ ] I have rebased against main before asking for a review

### CODE REVIEW

File: apps/service-portal/src/components/ContentBreadcrumbs/ContentBreadcrumbs.tsx [hunk 1]
```diff
@@ -118,7 +126,9 @@ const ContentBreadcrumbs: FC<React.PropsWithChildren<unknown>> = () => {
                   ? formatMessage(m.overview)
                   : formatMessage(item.name)}
               </Link>
-            ) : null,
+            ) : (
+              <GoBack noUnderline={true} display={['inline', 'inline']} />
```
Comment (thordurhhh) [inline apps/service-portal/src/components/ContentBreadcrumbs/ContentBreadcrumbs.tsx:130]: 👀 
```suggestion
              <GoBack noUnderline={true} display="inline" />
```

File: libs/service-portal/air-discount/src/lib/navigation.ts [hunk 2]
```diff
@@ -10,4 +10,5 @@ export const airDiscountNavigation: PortalNavigationItem = {
     icon: 'airplane',
     type: 'outline',
   },
+  children: [{ name: m.delegation, path: AirDiscountPaths.AirDiscountRoot }],
```
Comment (thordurhhh) [inline libs/service-portal/air-discount/src/lib/navigation.ts:13]: delegation?

Hide from breadcrumbs?

File: libs/service-portal/occupational-licenses/src/lib/navigation.ts [hunk 3]
```diff
@@ -11,6 +11,10 @@ export const occupationalLicensesNavigation: PortalNavigationItem = {
   },
   description: m.occupationalLicensesNavIntro,
   children: [
+    {
+      name: om.myLicenses,
+      path: OccupationalLicensesPaths.OccupationalLicensesRoot,
```
Comment (thordurhhh) [inline libs/service-portal/occupational-licenses/src/lib/navigation.ts:16]: hide from breadcrumbs?

File: libs/service-portal/information/src/components/PersonalInformation/Forms/ProfileForm/components/Inputs/BankInfoForm.tsx [hunk 4]
```diff
@@ -219,7 +219,7 @@ export const BankInfoForm: FC<React.PropsWithChildren<Props>> = ({
               {formatMessage(msg.buttonAccountSave)}
             </FormButton>
           )}
-          {loading && <LoadingDots />}
+          {loading && <SkeletonLoader />}
```
Comment (thordurhhh) [inline libs/service-portal/information/src/components/PersonalInformation/Forms/ProfileForm/components/Inputs/BankInfoForm.tsx:222]: Personally i would prefer to have the loading dots here 😭 
But OK!

File: libs/service-portal/documents/src/screens/Overview/Overview.tsx [hunk 5]
```diff
@@ -311,17 +312,11 @@ export const ServicePortalDocuments = () => {
...
-                size="small"
-                type="button"
-                variant="text"
-                truncate
-                onClick={() => navigate('/')}
-              >
-                {formatMessage(m.goBackToDashboard)}
-              </Button>
+              <GoBack
+                display={['inline', 'inline']}
```
Comment (thordurhhh) [inline libs/service-portal/documents/src/screens/Overview/Overview.tsx:316]:
```suggestion
                display="inline"
```

File: libs/service-portal/core/src/components/GoBack/GoBack.tsx [hunk 6]
```diff
@@ -0,0 +1,82 @@
...
+import * as styles from './GoBack.css'
+
+type GoBackProps = {
+  display?: ResponsiveProp<
+    'none' | 'flex' | 'block' | 'inline' | 'inlineBlock' | 'inlineFlex'
+  >
+  noUnderline?: boolean
+  truncate?: boolean
+  marginBottom?: ResponsiveProp<
+    | 0
```
Comment (thordurhhh) [inline libs/service-portal/core/src/components/GoBack/GoBack.tsx:14]: You could import `ResponsiveSpace` from `'@island.is/island-ui/core'` 

And then do: `marginBottom?: ResponsiveSpace`

**Review by thordurhhh — CHANGES_REQUESTED** [review]
Just a few minor things.

File: libs/service-portal/occupational-licenses/src/lib/navigation.ts (hunk 3 above)
Comment (Tryggvig) [reply libs/service-portal/occupational-licenses/src/lib/navigation.ts:16]: image

Comment (thordurhhh) [reply libs/service-portal/occupational-licenses/src/lib/navigation.ts:16]: ahhhh my bad. looks good 💯

**Review by thordurhhh — APPROVED** [review]
🚀

**Pull request merged**